scripts/jenkins/ardana/manual/deploy-crowbar.sh

```diff
@@ -20,7 +20,7 @@ source lib.sh
 
 validate_input
 setup_ansible_venv
-mitogen_disable
+mitogen_enable
```

Comment:
Why was it disabled before? And why enable it now? Could you comment?

Reply:
Hi, I just overtake it from Lumir's repo an I didn't pay attention into this. later he told me that he had got some timeout issues. The issue with timeouts seems to be related to `wait_for_connection` which is not reliable when using jumphost. See comment from Flavio in [ardana part](https://github.com/SUSE-Cloud/automation/blob/master/scripts/jenkins/ardana/ansible/bootstrap-vcloud-nodes.yml#L80).

Reply:
Thanks!